src/backend/launcher.ts

```diff
@@ -357,14 +359,34 @@ async function prepareWineLaunch(
     if (runner === 'legendary') {
       await legendarySetup(appName)
     }
-    if (
-      GlobalConfig.get().getSettings().experimentalFeatures
-        ?.automaticWinetricksFixes !== false
-    ) {
+    if (experimentalFeatures?.automaticWinetricksFixes !== false) {
       await installFixes(appName, runner)
     }
   }
 
+  if (runner === 'gog' && experimentalFeatures?.cometSupport !== false) {
+    if (isOnline() && !(await isInstalled('comet_dummy_service'))) {
+      await download('comet_dummy_service')
+    }
+    const installerScript = join(
+      runtimePath,
+      'comet_dummy_service',
+      'install-dummy-service.bat'
+    )
+    if (existsSync(installerScript)) {
+      await runWineCommand({
+        commandParts: [],
+        gameSettings,
+        protonVerb: 'runinprefix', // Prefix was verified before this, it's safe
+        skipPrefixCheckIKnowWhatImDoing: true
```

Comment:
Is there any reason you're skipping this here? Note that the function won't automatically run `wineboot -i` if this is not set, it'll first check whether prefix files are actually missing

Reply:
we are verifying the prefix before this, I think it's safe

Reply:
it's probably safe, but it just seems unnecessary to skip it

Reply:
right, let's remove it